scripts/analysis.py

```diff
@@ -999,43 +967,45 @@ def stacked_bar_chart(dictionary, timestep,
             label = str(key)
         # very first country does not have a 'bottom' argument
         if sum(dictionary[key]) == 0:
-            print(str(dictionary[key]) + 'has no values')
+            print(label + ' has no values')
         elif top_index is True:
-            plot = plt.bar(left=init_year + (timestep/12),
+            plot = plt.bar(left=init_year + (timestep / 12),
```

Comment:
I have made changes to the first paragraph. Sorry!

For the label, if I'm not mistaken the labels are for the legend. Maybe I don't quite understand your question. What labels are you referring to?